permission-portal-frontend/client/src/screens/ManageTeams.js

```diff
@@ -153,8 +153,6 @@ const ManageTeamsBase = observer((props) => {
                       value={!data.disabled ? 'active' : 'deactivated'}
                       onChange={(e) => {
                         props.store.updateUserByEmail(data.email, { disabled: e.target.value == 'deactivated' })
-                        data.disabled = e.target.value == 'deactivated'
```

Comment:
does the styling correctly update without this?

Reply:
Yes, the text is still bold when active and not bold when deactivated